internal/datasource/config/config_writer.go

```diff
@@ -82,23 +89,29 @@ func (cw *ConfigWriter) Rollback(ctx context.Context, skippedFiles CacheContent,
 	return nil
 }
 
-func (cw *ConfigWriter) Write(ctx context.Context, filesURL,
-	tenantID, instanceID string,
+// has cognitive-complexity of 11 due to the number or nil and error checks
+// nolint: revive
+func (cw *ConfigWriter) Write(ctx context.Context,
+	request *v1.ManagementPlaneRequest_ConfigApplyRequest, instanceID string,
 ) (skippedFiles CacheContent, err error) {
-	slog.DebugContext(ctx, "Write nginx config", "files_url", filesURL)
+	slog.DebugContext(ctx, "Write nginx config")
 	currentFileCache, skippedFiles := make(CacheContent), make(CacheContent)
 
 	cacheContent, err := cw.fileCache.ReadFileCache(ctx)
 	if err != nil {
-		return nil, err
+		slog.Warn("Unable to read file cache")
```

Comment:
```suggestion
		slog.WarnContext(ctx, "Unable to read file cache", "error", err)
```

Reply:
I'm curious, why are we no longer returning an error in this case?